tests/ui/bound-struct-without-generic.stderr

```diff
@@ -0,0 +1,5 @@
+error: #[error(bound = ...)] requires generics to apply bounds against
```

Comment:
Is it intentional that this is a different message than in the enum case in bound-enum-without-generic.stderr? It wasn't clear to me why they would be handled differently.

Reply:
That was unintentional; I can update it alongside any other changes you'd request, in the next iteration.

Reply:
Resolved in 02012b0.